src/main/jbake/content/blog/2015/06/17/marsjug-and-rivieradev-retrospective.adoc

```diff
@@ -0,0 +1,78 @@
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+++++
+
+I was well surprised to see the room packed for my presentation, as the RivieraDEV audience was mostly comprised of
+developers, not system administrators. This is a good sign: more and more, monitoring tools draw attention
+of the development side.
+
+++++
+<blockquote class="twitter-tweet" data-partner="tweetdeck"><p lang="fr" dir="ltr">Tenir ses applications à l&#39;œil avec <a href="https://twitter.com/hawkular_org">@hawkular_org</a> Metrics et <a href="https://twitter.com/tsegismont">@tsegismont</a> ! <a href="http://t.co/wul0NbeyTO">pic.twitter.com/wul0NbeyTO</a></p>&mdash; RivieraDEV (@RivieraDEV) <a href="https://twitter.com/RivieraDEV/status/609274503382831104">June 12, 2015</a></blockquote>
+<script async src="//platform.twitter.com/widgets.js" charset="utf-8"></script>
+++++
+
+I came back home with very good feedback from the attendees.
+
+== Interoperability idea validated
+
+One of the nicest questions I was asked at RivieraDEV was, in the following or similar terms:
+
+____
+I'm interested but I have Nagios everywhere, and I won't remove it. Can I use Hawkular with it?
+____
+
+While we haven't worked on Nagios interoperability yet, we have a tool called
+https://github.com/hawkular/hawkular-metrics/tree/master/clients/ptranslator[PTrans], which listens to metric data sent
+over several common protocols (_graphite_ and _collectd_ among others).
+
+So Nagios integration is possible, and would be a very nice contribution (subliminal message).
+
+== Dashboards
+
+It's one thing to store data, but of course everyone wants to do something with it! So the ability to work with Grafana
+on top of Hawkular Metrics was appreciated. And as promised to a participant,
+there's now http://www.hawkular.org/docs/components/metrics/grafana_integration.html[a guide for the setup].
+
+But we're also working on our own charting Javascript library, so you'll be soon be able to create your own dashboards
+or embed graphs in your applications.
+
+== OpenTSDB comparison
+
+At the MarsJUG as well as RivieraDEV, I was asked how Hawkular Metrics relates to OpenTSDB, and why we don't use
+OpenTSDB.
+
+Here's my own opinion:
+
+. It is similar in goals, I think: being simple to use (from a client), and capable of ingesting and storing
+loads of data.
+
+. With the RHQ project, we have experience in storing metric data in Apache Cassandra; it fits our known needs, so from
+a user/customer support perspective, it feels more comfortable to me.
+
+. Hawkular Metrics runs on Microsoft Windows out-of-the-box; as far as I know, OpenTSDB does not;
```

Comment:
How about 'Hawkular Metrics can run on Microsoft Windows'. The way it is worded now, it sounds to me like it is a Windows product ;)